packages/payload/src/auth/crypto.ts

```diff
@@ -1,11 +1,12 @@
-// @ts-strict-ignore
 import crypto from 'crypto'
 
 const algorithm = 'aes-256-ctr'
 
 export function encrypt(text: string): string {
   const iv = crypto.randomBytes(16)
-  const cipher = crypto.createCipheriv(algorithm, this.secret, iv)
+  // @ts-expect-error - vestiges of when tsconfig was not strict. Feel free to improve
+  const secret = this.secret
+  const cipher = crypto.createCipheriv(algorithm, secret, iv)
```

Comment:
Since I think it might be useful to someone in the future, I'm sharing the reasoning here as a safety tip: extracting the conflicting variable to a new line is intentional. I tend to do this because if another error appears on the line below, `@ts-expect-error` would silence them all.